libvast/vast/system/exporter.hpp

```diff
@@ -66,7 +66,7 @@ struct exporter_state {
   std::chrono::system_clock::time_point start = {};
 
   /// Stores various meta information about the progress we made on the query.
-  query_status query = {};
+  query_status status = {};
```

Comment:
```suggestion
  struct query_status query_status = {};
```

Gives you the best of both worlds and a lot of clarity at all use sites when reading the code.